core/src/test/java/dev/streamx/cli/ingestion/IngestionArgumentsValidationTest.java

```diff
@@ -58,50 +51,7 @@ public void shouldRejectIngestionWithMissingParameter(QuarkusMainLauncher launch
         Usage: streamx publish [-hV] [[--ingestion-url=<propagateIngestionUrl>]]
                                (<channel> <key> [payloadFile]) [[[-s=<string> |
                                -b=<binary> | -j=<json>]]...]
-        Send publication data
-              <channel>       Channel that message will be published to
-              <key>           Message key
-              [payloadFile]   File containing the payload to be published.
-                              This is an optional parameter.
-                              This parameter is equivalent to
-                              -j=file://<payloadFile> specified before other Payload
-                                Defining Option.
-                
-                              If this parameter is present, it has the highest priority
-                                in defining the payload.
-          -h, --help          Show this help message and exit.
-              --ingestion-url=<propagateIngestionUrl>
-                              Address of 'rest-ingestion-service'
-                                Default: http://localhost:8080
-          -V, --version       Print version information and exit.
-                
-        Payload Defining Options:
-            Payload can be defined by specifying an explicit full payload.
-            For example, publish -j "{}" (...)
-            will send {} to the ingestion service.
-                
-            A payload can also be created by specifying one or more
-            JSONPath expressions with payload fragments.
-            For example, publish -s type=string -s content.bytes=hello (...)
-            will send {"type":"string","content":{"bytes":"hello"}}
-            to the ingestion service.
-                
-            There are few ways to define the parameter value:
-            * if value has prefix file:// then value will be loaded
-              from file with given (relative or absolute) path
-            * otherwise raw value will be used as value
-                
-            Payload fragment type:
-            * if the value is a string fragment, use -s option
-            * if the value is a JSON fragment, use -j option
-            * if the value is a binary value fragment use -b option
-                
-          -b, --binary-fragment=<binary>
-                              Defines a payload fragment of binary type
-          -j, --json-fragment=<json>
-                              Defines a payload fragment of JSON node type
-          -s, --string-fragment=<string>
-                              Defines a payload fragment of string type""");
+        Try 'streamx publish --help' for more information.""");
```

Comment:
If it's possible - add newline to separate suggestion "Try" from rest of information.

Reply:
Done